test/e2e/files/crio-nri-enable

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+mkdir -p /etc/crio
+
+cat >> /etc/crio/crio.conf <<EOF
+# CRI-O NRI configuration.
+[crio.nri]
+# Globally enable or disable NRI.
+enable_nri = true
+# NRI configuration file to use.
+# nri_config_file = "/etc/nri/nri.conf"
+# NRI socket path to create.
+# nri_socket_path = "/var/run/nri/nri.sock"
+# NRI plugin directory to use.
+# nri_plugin_dir = "/opt/nri/plugins"
+EOF
```

Comment:
I think it would be better to either use command line options to generate an updated config file
```
CRIO_CONF=/etc/crio/crio.conf
cp $CRIO_CONF $CRIO_CONF.orig
crio --enable-nri config > $CRIO_CONF
```

or use a drop-in like

```
mkdir -p /etc/crio/crio.conf.d
cat > /etc/crio/crio.conf.d/10-enable-nri.conf <<EOF
[crio.nri]
enable_nri = true
EOF
```